docs/source/filters.rst

```diff
@@ -73,3 +73,4 @@ Interactive Demo
 .. raw:: html
     :file: gradio/edge_filter_gradio.html
     :file: gradio/filtering_operators_gradio.html
+    :file: gradio/edge_detector_gradio.html
```

Comment:
I don't think including multiple `:file:` entries under the HTML directive will work. I would suggest putting 

```
.. raw:: html
    :file: gradio/edge_detector_gradio.html
```

Just directly inside the "Edge Detection" section instead of here at the bottom. We can move the edge_filter_gradio and filtering_operators_gradio ones as well in a different PR.